src/library/helpers/make-teacher-edition-links.js

```diff
@@ -0,0 +1,19 @@
+// Selector should look like '#teacher-edition-links a'
+//
+var MakeTeacherEditionLinks = function (selector) {
+  var updateAnchorTag = function (anchor) {
+    var oldLink = anchor.getAttribute('href')
+    var domain = encodeURIComponent('https://learn.staging.concord.org')
```

Comment:
I know I said the domain would be OK to be hard coded. But with the approach above it wouldn't be possible to use this function on both staging and production. 

Here are a couple of options:
1. pass domain as another parameter to MakeTeacherEditionLinks  this way the collection page can use learn.staging.concord.org or it could use learn.concord.org
2. figure out the domain based on the current pages URL.  It would probably be best to combine this with option 1.  So if the domain parameter is not set then the function will use the current page's domain.  This way the example page above can hard code the domain to be learn.staging.concord.org.  But a collection page that is on learn.concord.org or learn.staging.concord.org wouldn't need specify the domain.

Reply:
Ooff.  Yeah.  

I guess I was in a hurry.  Thanks again.